dashboard/db/migrate/20170522221144_add_course_id_to_sections.rb

```diff
@@ -0,0 +1,5 @@
+class AddCourseIdToSections < ActiveRecord::Migration[5.0]
+  def change
+    add_column :sections, :course_id, :integer
```

Comment:
We shouldn't need an index. I will move it to be adjacent, and use `t.references`